pom.xml

```diff
@@ -80,14 +80,14 @@
         <javassist.version>3.19.0-GA</javassist.version>
         <javax-activation.version>1.1.1</javax-activation.version>
         <jcs.version>2.1</jcs.version>
-        <jersey.version>2.25</jersey.version>
+        <jersey.version>2.22.2</jersey.version>
```

Comment:
Going backwards in the jersey version? We would need to also update the NOTICES file.